spec/models/mention_spec.rb

```diff
@@ -1,21 +1,13 @@
 require "rails_helper"
 
 RSpec.describe Mention, type: :model do
-  let(:user)        { create(:user) }
-  let(:article)     { create(:article, user_id: user.id) }
-  let(:comment)     { create(:comment, user_id: user.id, commentable_id: article.id) }
+  let(:comment) { create(:comment, commentable: create(:podcast_episode)) }
 
-  it "calls on Mentions::CreateAllJob" do
-    described_class.create_all(comment) do
-      expect(Mentions::CreateAllJob).to have_received(:perform_later).with(comment.id, "Comment")
+  describe "#create_all" do
+    it "enqueues a job to create mentions" do
+      assert_enqueued_with(job: Mentions::CreateAllJob, args: [comment.id, "Comment"], queue: "mentions_create_all") do
+        described_class.create_all(comment)
+      end
     end
   end
-
-  it "creates a valid mention" do
```

Comment:
Not sure why should these tests be removed.

Reply:
I thought about it and I decided to remove it because it's not actually testing anything related to our code, it's testing FactoryBot and Rails. 

I don't think it's an absolute but I like the general rule in testing of trusting the framework and not testing its default behavior.

Reply:
Actually, here the first test is meant to be checking that we specified all of the necessary attributes for the factory so that it could be used in the tests without issues. However, I don't see it as a huge problem if we remove it.
The second test was written as a part of this pr: #3188 Unfortunately, I don't remember the exact problem but the validation was failing with an exception in some cases, so I added a test to check that it shouldn't fail in that case anymore

Reply:
I went ahead and put these two specs back in because it seems that it was there for a good reason. Perhaps they should be replaced by validation specs in the near future.